### PR TITLE
Fix no-blake2 for Windows classic build

### PR DESCRIPTION
**Pull request by matbech**
Fix no-blake2 for Windows classic build

### CODE REVIEW

**Comment by kroeckx** [comment]
@levitte: Care to look at this?

File: util/mk1mf.pl [hunk 1]
```diff
@@ -284,6 +285,7 @@
 $cflags.=" -DOPENSSL_NO_MD4"  if $no_md4;
 $cflags.=" -DOPENSSL_NO_MD5"  if $no_md5;
 $cflags.=" -DOPENSSL_NO_RMD160" if $no_ripemd;
+$cflags.=" -DOPENSSL_NO_BLAKE2" if $no_blake2;
```
Comment (levitte) [inline util/mk1mf.pl]: This is not necessary, it's defined in opensslconf.h

**Comment by levitte** [comment]
Other than that comment, it looks fine to me

File: util/mk1mf.pl (hunk 1 above)
Comment (richsalz) [inline util/mk1mf.pl]: Perhaps some files don’t include opensslconf.h that should?

Comment (matbech) [inline util/mk1mf.pl]: Corrected. Thanks.

**Comment by richsalz** [comment]
done, thanks!

**Pull request closed without merging**